internal/schema/schema.go

```diff
@@ -184,21 +232,27 @@ func (s *Schema) QueryFieldsForTypeName(name string, maxDepth int) string {
 	return t.GetQueryStringFields(s, 0, maxDepth)
 }
 
-// QueryArgs is meant to fill in the data necessary for a query(<args_go_here>)
-// string.  For example, query($guids: [String]!) { actor ...
-func (s *Schema) QueryArgs(t *Type, fields []string) []QueryArg {
+// ArgumentsForEndpoint is meant to fill in the data necessary for a query(<args_go_here>)
+// string.  i.e: query($guids: [String]!) { actor ...
+func (s *Schema) ArgumentsForEndpoint(t *Type, fields []string, includeNullable bool) []QueryArg {
```

Comment:
Thoughts on naming this `BuildQueryStringArguments()`? Not a blocker by any means.

Reply:
I renamed, hows that?